lib/errors/aggregate-error.ts

```diff
@@ -0,0 +1,33 @@
+import BaseError from './base-error';
+
+/**
+ * A wrapper for multiple Errors
+ *
+ * @param {Error[]} [errors] Array of errors
+ *
+ * @property errors {Error[]}
+ */
+class AggregateError extends BaseError {
+  errors: Array<AggregateError | Error>;
+
+  constructor(errors: Array<AggregateError | Error>) {
+    super('');
```

Comment:
If we were to declare the BaseError message optional, then this could be an empty super call